src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -5,82 +5,62 @@
 
```

Comment:
Stuff in this file is useful as a preview of a working implementation, but not in need of deep review at this point.